tests/containers/dotnet/dotnet.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
Is there any reason to keep the autogenerated files here? If we can, keep only the actual source (i.e `Fibonacci.cs`) and avoid committing extra files that need to be CRed and maintained (instead of simply being autogenerated correctly during tests run)